### PR TITLE
[REEF-761] Implement ActiveContext.SendMessage()

### PR DESCRIPTION
**Pull request by afchung**
JIRA:
  [REEF-761] https://issues.apache.org/jira/browse/REEF-761

Pull Request:
  Closes #

### CODE REVIEW

File: lang/cs/Org.Apache.REEF.Driver/Bridge/Events/ActiveContext.cs [hunk 1]
```diff
@@ -94,7 +94,7 @@ public void SubmitContextAndService(IConfiguration contextConfiguration, IConfig
 
         public void SendMessage(byte[] message)
```
Comment (markusweimer) [inline lang/cs/Org.Apache.REEF.Driver/Bridge/Events/ActiveContext.cs]: Maybe we need a comment here to tell users that this won't work with C# Evaluators? Also, maybe link to the JIRA for it.

**Comment by afchung** [comment]
@markusweimer Thanks for the review! I've addressed your comment, please have another look.

**Comment by markusweimer** [comment]
I'm +1 on merging this. However, I can't test due to a hardware failure on the road. Any volunteers?

**Comment by dongjoon-hyun** [comment]
I will test and merge this, @markusweimer .  :)

**Pull request closed without merging**